hackathon_site/hardware/models.py

```diff
@@ -29,6 +29,7 @@ class Meta:
     max_per_team = models.IntegerField(null=True)
     picture = models.ImageField(upload_to="uploads/hardware/pictures/", null=False)
     categories = models.ManyToManyField(Category)
+    objects = AnnotatedHardwareManager()
```

Comment:
I would put this at the top, above the `class Meta` and with a blank line between them. From a code organization perspective, it at least shouldn't be mixed in between database fields.